sky/serve/controller.py

```diff
@@ -42,13 +44,17 @@ def __init__(self, service_name: str, service_spec: serve.SkyServiceSpec,
                  task_yaml: str, port: int) -> None:
         self._service_name = service_name
         self._replica_manager: replica_managers.ReplicaManager = (
-            replica_managers.SkyPilotReplicaManager(service_name=service_name,
-                                                    spec=service_spec,
-                                                    task_yaml_path=task_yaml))
+            replica_managers.SkyPilotReplicaManager(
+                service_name=service_name,
+                spec=service_spec,
+                initial_version=constants.INITIAL_VERSION,
```

Comment:
Do we really need this argument? It seems we are passing the constant all over the places.

Reply:
see above

Reply:
removed